static/app/components/tooltip.tsx

```diff
@@ -320,67 +258,44 @@ function Tooltip({
-    },
-  };
-
-  const visible = forceShow || isOpen;
-
-  const tip = visible ? (
-    <Popper placement={position} modifiers={modifiers}>
-      {({ref, style, placement, arrowProps}) => (
-        <PositionWrapper style={style}>
-          <TooltipContent
-            id={tooltipId}
-            initial={{opacity: 0}}
-            animate={{
-              opacity: 1,
-              scale: 1,
-              transition: testableTransition({
-                type: 'linear',
-                ease: [0.5, 1, 0.89, 1],
-                duration: 0.2,
-              }),
-            }}
-            exit={{
-              opacity: 0,
-              scale: 0.95,
-              transition: testableTransition({type: 'spring', delay: 0.1}),
-            }}
-            style={computeOriginFromArrow(position, arrowProps)}
-            transition={{duration: 0.2}}
-            className="tooltip-content"
-            aria-hidden={!visible}
-            ref={ref}
-            data-placement={placement}
-            popperStyle={popperStyle}
-            onMouseEnter={() => isHoverable && handleOpen()}
-            onMouseLeave={() => isHoverable && handleClose()}
-          >
-            {title}
-            <TooltipArrow
-              ref={arrowProps.ref}
-              data-placement={placement}
-              style={arrowProps.style}
-            />
-          </TooltipContent>
-        </PositionWrapper>
-      )}
-    </Popper>
-  ) : null;
+  // The tooltip open/closed state can be controlled from the forceShowProp
+  const isOpenOrControlled = typeof forceShow === 'boolean' ? forceShow : isOpen;
 
   return (
     <Manager>
       <Reference>{({ref}) => renderTrigger(children, ref)}</Reference>
-      {createPortal(<AnimatePresence>{tip}</AnimatePresence>, tooltipPortal)}
+      {createPortal(
+        <AnimatePresence>
+          {isOpenOrControlled ? (
+            <Popper placement={position} modifiers={modifiers}>
+              {({ref, style, placement, arrowProps}) => (
+                <PositionWrapper style={style}>
+                  <TooltipContent
+                    id={tooltipId}
+                    data-placement={placement}
+                    style={computeOriginFromArrow(position, arrowProps)}
+                    className="tooltip-content"
+                    aria-hidden={false}
+                    ref={ref}
+                    popperStyle={popperStyle}
+                    onMouseEnter={() => isHoverable && handleOpen()}
+                    onMouseLeave={() => isHoverable && handleClose()}
+                    {...TOOLTIP_ANIMATION}
+                  >
+                    {title}
+                    <TooltipArrow
+                      ref={arrowProps.ref}
+                      data-placement={placement}
+                      style={arrowProps.style}
+                    />
+                  </TooltipContent>
+                </PositionWrapper>
+              )}
+            </Popper>
+          ) : null}
```

Comment:
```suggestion
          {!isOpenOrControlled ? null : (
            <Popper placement={position} modifiers={modifiers}>
              {({ref, style, placement, arrowProps}) => (
                <PositionWrapper style={style}>
                  <TooltipContent
                    id={tooltipId}
                    data-placement={placement}
                    style={computeOriginFromArrow(position, arrowProps)}
                    className="tooltip-content"
                    aria-hidden={false}
                    ref={ref}
                    popperStyle={popperStyle}
                    onMouseEnter={() => isHoverable && handleOpen()}
                    onMouseLeave={() => isHoverable && handleClose()}
                    {...TOOLTIP_ANIMATION}
                  >
                    {title}
                    <TooltipArrow
                      ref={arrowProps.ref}
                      data-placement={placement}
                      style={arrowProps.style}
                    />
                  </TooltipContent>
                </PositionWrapper>
              )}
            </Popper>
          )}
```

I've been doing this recently and I think it makes it a tad more visually understandable